API Project/src/main/java/hello/ItemController.java

```diff
@@ -0,0 +1,32 @@
+package hello;
+
+import java.util.concurrent.atomic.AtomicLong;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import java.util.ArrayList;
+import java.util.Date;
+@RestController
+public class ItemController{
+
+private final AtomicLong counter = new AtomicLong();
+private ArrayList<CulturalHeritageItem> items = new ArrayList<>();
+
+@RequestMapping("/get-items")
+  public ArrayList<CulturalHeritageItem> getItems() {
+    return this.items;
+  }
+
+@RequestMapping("/add-item")
+  public CulturalHeritageItem addItem(	@RequestParam(value="title", defaultValue="") String title,
+					@RequestParam(value="description", defaultValue="") String description,
+					@RequestParam(value="continent", defaultValue="") String continent,
+					@RequestParam(value="city", defaultValue="") String city
+					) {
+    CulturalHeritageItem item = new CulturalHeritageItem(counter.incrementAndGet(),new User(), title, description, continent, city,new Date());
```

Comment:
It's often considered as best practice to pass object references like this:
`POST /users/:id/cultural-heritages`
So if I do a request like this:
```
POST /users/3/cultural-heritages
{
    "city": "Istanbul",
    ...
}
```
It creates the given cultural heritage item for user with the id of 3